integration/local-gadget/top_ebpf_test.go

```diff
@@ -62,12 +60,8 @@ func TestTopEbpf(t *testing.T) {
 	}
 
 	commands := []*Command{
-		CreateTestNamespaceCommand(ns),
 		topEbpfCmd,
 		SleepForSecondsCommand(2), // wait to ensure local-gadget has started
```

Comment:
We can remove `SleepForSecondsCommand` from here because of `StartAndStop` logic but the idea was to ensure we are ready for tracing before starting a container otherwise we can have weird issues like missing some [initial events](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/integration/local-gadget/trace_exec_test.go#L30) for a [test](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/integration/local-gadget/trace_exec_test.go#L40).